model/execution/DeliveryExecutionList.php

```diff
@@ -128,6 +131,42 @@ private function createExecution($cachedData, $extraFields)
         return $execution;
     }
 
+    private function getAdjustedTime(string $deliveryExecutionId): ?int
+    {
+        $deliveryExecution = $this->getDeliveryExecutionManagerService()->getDeliveryExecutionById($deliveryExecutionId);
+        $testSession = $this->getTestSessionService()->getTestSession($deliveryExecution);
+        $adjustedTime = null;
```

Comment:
In response to `/taoProctoring/Monitor/deliveryExecutions` we return 0 if time was not adjusted. Maybe it would be better to use here 0 as default value instead of `null`?

Reply:
yes, sure, just wanted to note the third state when noone touched the timer

Reply:
changed